packages/fx-core/src/plugins/resource/bot/utils/retryHandler.ts

```diff
@@ -13,7 +13,8 @@ export class RetryHandler {
         return response;
       } catch (e) {
         if (retries <= 0) {
-          throw e;
+          if (ignoreError) break;
```

Comment:
I would like as below to let it retry even ignore error
```
if (!ignoreError) {
  throw e;
}
```

Reply:
only if(retries <= 0) then it will judge throw or break; anyway, i update the code to make it cleaner